contracts/modules/RelayerManager.sol

```diff
@@ -368,4 +386,31 @@ contract RelayerManager is BaseFeature {
         // solhint-disable-next-line no-inline-assembly
         assembly { chainId := chainid() }
     }
+
+    function getRequiredSignaturesToStartSession(address _wallet) internal view returns (uint256, OwnerSignature) {
+        // owner  + [n/2] guardians
+        uint numberOfGuardians = Utils.ceil(guardianStorage.guardianCount(_wallet), 2);
+        require(numberOfGuardians > 0, "RM: no guardians set on wallet");
+        uint numberOfSignatures = 1 + numberOfGuardians;
+        return (numberOfSignatures, OwnerSignature.Required);
+    }
+
+    function startSession(address _wallet, bytes calldata _data) internal returns (bool) {
+        require(_data.length >= 68, "RM: Invalid session parameters");
+        (address key, uint64 expires) = abi.decode(_data[36:], (address, uint64));
+        if (key != address(0)) {
+            if (expires > 0) {
+                sessions[_wallet] = Session(key, expires);
```

Comment:
We are overwriting any existing sessions here. Might be better to throw instead and require owner to explicitly clear existing valid session before replacing?